lib/ci/lib/rails_upgrade_analyzer/job_result.rb

```diff
@@ -0,0 +1,40 @@
+module RailsUpgradeAnalyzer
+  class JobResult
+
+    attr_reader :deprecations, :description, :job_number, :tests, :passed, :failures, :errors
+
+    def initialize(job_number, options={})
+      @job_number = job_number
+      @description = options[:description]
+      @tests = options.fetch(:tests, 0).to_i
+      @passed = options.fetch(:passed, 0).to_i
+      @failures = options.fetch(:failures, 0).to_i
+      @errors = options.fetch(:errors, 0).to_i
+      @deprecations = options.fetch(:deprecations, Hash.new(0))
+    end
+
+    def deprecation_count
+      deprecations.each_value.inject(&:+) || 0
```

Comment:
`deprecations.values.inject(&:+)` seems a little more natural, if I understand the point of this code...